### ISSUE TITLE
Unknown Time Zone

### ISSUE THREAD

**Issue by Dark-Sol** [mlange-42/tom#40]
When trying to get the forecast for an assortment of North American cities/towns, I get an error saying that the time zone for that city is unknown:

```
F:\tui>tom Halifax                                                                                                                                                                                                                                                                                                         
ERROR: unknown time zone America/Halifax

Run `tom -h` for help!
```

**Comment by mlange-42** [comment]
Many thanks for reporting, @Dark-Sol. I will look into it.

**Comment by mlange-42** [comment]
@Dark-Sol Do you get a location dialog like this before the error? And does it work for you for other, e.g. European cities?

![grafik](https://github.com/user-attachments/assets/d2bf1e20-1a69-4044-9f40-889fcb412792)

**Comment by Dark-Sol** [comment]
@mlange-42: I do get that location dialog before the error. I tried looking up the weather for London, UK but received the same error.
![image](https://github.com/user-attachments/assets/5e44a738-5a97-488e-b263-35dee0997194)

**Comment by Dark-Sol** [comment]
Oh! As an addendum, I _am_ trying to run this on Windows 10. Maybe that has something to do with it?

**Comment by mlange-42** [comment]
Thanks @Dark-Sol!

I think I found the problem. The respective time zones are not available on your system. However, there is the Go package `time/tzdata` that ptovides them when imported.

Could you give the fix a try, please?

If you have Go installed, you can install the fixed version with

```
go install github.com/mlange-42/tom@fix-time-zone
```

Alternatively, you can download the binaries from the latest CI run: https://github.com/mlange-42/tom/actions/runs/12313852956
(scroll down to the bottom, section "Artifacts").

**Comment by Dark-Sol** [comment]
The new binary worked! :)

**Comment by mlange-42** [comment]
Ok, nice! I will prepare a new release with the fix. Thanks again for reporting!

**Comment by mlange-42** [comment]
Fix released with v0.5.0.
